monetio/sat/_modis_l2_mm.py

```diff
@@ -2,6 +2,7 @@
 import sys
 from collections import OrderedDict
 from glob import glob
+from datetime import datetime, timezone
```

Comment:
please the formatter
```suggestion
from datetime import datetime, timezone
from glob import glob
```